templates/section_project/contact_manager.html

```diff
@@ -0,0 +1,145 @@
+            {
+                "title": "More info",
+                "content": "<div>Enter the information for the delivery manager. This is a person who can be contacted in regards to discussing the project.</div><br>"
+            }
+        ]
+    }) }}
+    <form action="{{ url_for('survey') }}" method="GET">
+        {{ onsLabel({
+        "id": 'contact-email-label',
+        "for": "contact-email",
+        "text": 'Delivery Manager Information',
+        "description": 'Enter the information of the delivery manager for this project.'
+    }) }}
+        {{ onsInput({
+        "id": "contact-email",
+        "type": "email",
+        "autocomplete": "email",
+        "label": {
+            "text": "Email address",
+            "description": "This will be stored and used as a point of contact for the project."
+
+        }
+
+    }) }}
+        {% call onsQuestion({
+        "classes": "ons-u-mt-no",
+        "legendIsQuestionTitle": true
+    }) %}
+        {{ onsLabel({
+        "id": 'contact-grade-label',
+        "for": "contact-grade",
+        "text": 'Delivery Manager Grade',
+        "description": 'Select the grade of the delivery manager.'
+    }) }}
+        {{ onsRadios({
+            "dontWrap": true,
+            "name": "role",
+            "radios": [
+                {
+                    "id": "Grade 6",
+                    "label": {
+                        "text": "Grade 6"
+                    },
+                    "value": "Grade 6"
+                },
+                                {
+                    "id": "Grade 7",
+                    "label": {
+                        "text": "Grade 7"
+                    },
+                    "value": "Grade 7"
+                },
+                {
+                    "id": "SEO",
+                    "label": {
+                        "text": "SEO"
+                    },
+                    "value": "SEO"
+                },
+                {
+                    "id": "HEO",
+                    "label": {
+                        "text": "HEO"
+                    },
+                    "value": "HEO"
+                },
+                {
+                    "id": "other",
+                    "label": {
+                        "text": "Other"
+                    },
+                    "value": "other",
+                    "other": {
+                        "id": "other-input",
+                        "name": "other",
+                        "label": {
+                            "text": "Enter your role"
+                        }
+                    }
+                }
+            ]
+        }) }}
+        {% endcall %}
+        {{ onsButton({
+        "id": 'save-values-button',
+        "text": "Save and continue",
+        "url": url_for('project'),
+        "attributes": {
+            "onclick": "storeData();"
+        }
+    }) }}
+        <script src="{{ url_for('static', filename='formScripts.js') }}"></script>
```

Comment:
I think there is probably a pattern where onclick redirects to a common route to store data, rather than have the function in each html file.  However, for now lets go with what we have.

Reply:
Noted 👍